src/transactions/0_send.js

```diff
@@ -36,19 +40,25 @@ import { getTimeWithOffset } from './utils/time';
 
 export default function send({
 	recipientId,
+	recipientPublicKey,
 	amount,
 	secret,
 	secondSecret,
 	data,
 	timeOffset,
 }) {
+	const { address, publicKey } = checkRecipientIdAndRecipientPublicKey({
```

Comment:
Since we go on to use the output of this function I think `check` is misleading. Maybe `getAddressAndPublicKey` or `getAddressAndPublicKeyFromRecipientData`?